<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportRecordPersonVisitsSetupDialog</class>
 <widget class="QDialog" name="ReportRecordPersonVisitsSetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>578</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lblBegDate">
       <property name="text">
        <string>Дата &amp;начала периода</string>
       </property>
       <property name="buddy">
        <cstring>edtBegDate</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="CDateEdit" name="edtBegDate">
       <property name="minimumSize">
        <size>
         <width>252</width>
         <height>0</height>
        </size>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblEndDate">
       <property name="text">
        <string>Дата &amp;окончания периода</string>
       </property>
       <property name="buddy">
        <cstring>edtEndDate</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="CDateEdit" name="edtEndDate">
       <property name="minimumSize">
        <size>
         <width>252</width>
         <height>0</height>
        </size>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="chkDetailChildren">
       <property name="text">
        <string>Детализация по подросткам</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="chkVisitHospital">
       <property name="text">
        <string>Учитывать посещения ДС</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lblEventPurpose">
       <property name="text">
        <string>&amp;Назначение обращения</string>
       </property>
       <property name="buddy">
        <cstring>cmbEventPurpose</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="CRBComboBox" name="cmbEventPurpose">
       <property name="minimumSize">
        <size>
         <width>252</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="lblEventType">
       <property name="text">
        <string>&amp;Тип обращения</string>
       </property>
       <property name="buddy">
        <cstring>cmbEventType</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="CRBComboBox" name="cmbEventType">
       <property name="minimumSize">
        <size>
         <width>252</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="lblOrgStructure">
       <property name="text">
        <string>&amp;Подразделение</string>
       </property>
       <property name="buddy">
        <cstring>cmbOrgStructure</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="COrgStructureComboBox" name="cmbOrgStructure">
       <property name="minimumSize">
        <size>
         <width>252</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="lblPerson">
       <property name="text">
        <string>&amp;Врач</string>
       </property>
       <property name="buddy">
        <cstring>cmbPerson</cstring>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="CPersonComboBoxEx" name="cmbPerson">
       <property name="minimumSize">
        <size>
         <width>252</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Врач</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="lblSex">
       <property name="text">
        <string>По&amp;л</string>
       </property>
       <property name="buddy">
        <cstring>cmbSex</cstring>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QComboBox" name="cmbSex">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>М</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ж</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="lblAge">
       <property name="text">
        <string>Во&amp;зраст с</string>
       </property>
       <property name="buddy">
        <cstring>edtAgeFrom</cstring>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QFrame" name="frmAge">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="_2">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSpinBox" name="edtAgeFrom">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximum">
           <number>150</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblAgeTo">
          <property name="text">
           <string>по</string>
          </property>
          <property name="buddy">
           <cstring>edtAgeTo</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="edtAgeTo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximum">
           <number>150</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblAgeYears">
          <property name="text">
           <string>лет</string>
          </property>
          <property name="buddy">
           <cstring>edtAgeTo</cstring>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>21</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="lblVisitPayStatus">
       <property name="text">
        <string>Флаг финансирования</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QComboBox" name="cmbVisitPayStatus">
       <property name="minimumSize">
        <size>
         <width>252</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>не задано</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>не выставлено</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>выставлено</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>отказано</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>оплачено</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="11" column="0" colspan="2">
      <widget class="QSplitter" name="splitterTree">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="CSocStatusTreeView" name="treeItems">
        <property name="minimumSize">
         <size>
          <width>204</width>
          <height>0</height>
         </size>
        </property>
       </widget>
       <widget class="CRBListBox" name="lstItems">
        <property name="minimumSize">
         <size>
          <width>204</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>COrgStructureComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgStructComboBoxes.h</header>
  </customwidget>
  <customwidget>
   <class>CPersonComboBoxEx</class>
   <extends>QComboBox</extends>
   <header>Orgs/PersonComboBoxEx.h</header>
  </customwidget>
  <customwidget>
   <class>CSocStatusTreeView</class>
   <extends>QTreeView</extends>
   <header>Registry/SocStatusComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>CRBListBox</class>
   <extends>QListView</extends>
   <header>library/RBListBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ReportRecordPersonVisitsSetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>583</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ReportRecordPersonVisitsSetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>583</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
